apps/app/dashboards/demography/immigration/index.tsx

```diff
@@ -24,14 +106,345 @@ const Immigration: FunctionComponent<ImmigrationProps> = ({}) => {
+                  }}
+                  type="state"
+                  color="purples"
+                />
+              </>
+            }
+          />
+        </Section>
+
+        {/* How many people from {{ country }} are entering and leaving the country? */}
+        <Section
+        // date={timeseries.data_as_of}
+        >
+          <>
+            <div className="w-full space-y-12">
+              <div className="w-full space-y-6">
+                <h4 className="text-center">
+                  {t("country_header", {
+                    country: data.country_label,
+                    context: data.country_label && "overall",
+                  })}
+                </h4>
+                <div className="flex flex-col items-center justify-center space-y-3">
+                  <div className="grid w-full grid-cols-12 lg:grid-cols-10">
+                    <div className="col-span-10 col-start-2 sm:col-span-8 sm:col-start-3 md:col-span-6 md:col-start-4 lg:col-span-4 lg:col-start-4">
+                      <ComboBox
+                        enableFlag
+                        imageSource="https://flagcdn.com/h20/"
+                        fallback={<GlobeAltIcon className="w-4.5 h-4.5 mx-auto text-black" />}
+                        placeholder={t("search_country")}
+                        options={COUNTRY_OPTIONS}
+                        selected={
+                          data.country
+                            ? COUNTRY_OPTIONS.find(e => e.value === data.country.value)
+                            : undefined
+                        }
+                        onChange={country => {
+                          setData("country", country);
+                          if (country) {
+                            get("/dashboard", {
+                              dashboard: "immigration_country",
+                              country: country.value,
+                            })
+                              .then(({ data }) => {
+                                if (data.timeseries_country && data.timeseries_country_callout) {
+                                  setData("timeseries_country", data.timeseries_country);
+                                  setData(
+                                    "timeseries_country_callout",
+                                    data.timeseries_country_callout
+                                  );
+                                }
+                                setData("country_label", country.label);
+                              })
+                              .catch(e => {
+                                console.error(e);
+                              });
+                          }
+                        }}
+                      />
+                    </div>
+                  </div>
+                  <p className="text-dim text-center text-sm">{t("country_desc")}</p>
+                </div>
+              </div>
```

Comment:
This design is off from figma.

Reply:
I changed the design slightly because the search was in an awkward position

Reply:
Yep, noted in the group chat. tyvm